rehive/api/resources/admin_resources.py

```diff
@@ -647,3 +718,81 @@ def __init__(self, client, endpoint, filters=None):
     @classmethod
     def get_resource_name(cls):
         return 'address'
+
+
+class APIAdminSearch(ResourceList):
+    @classmethod
+    def get_resource_name(cls):
+        return 'search'
+
+
+class APIAdminServices(ResourceList, ResourceCollection):
+    def __init__(self, client, endpoint='', filters=None):
+        self.resources = (APIAdminPermissions,)
+
+        super().__init__(client, endpoint, filters)
+
+    def create(self, name, url, **kwargs):
+        return super().create(name=name, url=url, **kwargs)
+
+    def put(self, name, url, **kwargs):
```

Comment:
I know that technically the PUT method requires all fields to be included. But I think it will be better to leave this to function like the PATCH and not require specific fields (as technically the API handles PUT requests in the same way as PATCH requests).

Reply:
Maybe apply the same to the other places where you added custom `put` methods down below.